core/arch/arm/include/arm64.h

```diff
@@ -305,6 +305,20 @@
 #define ID_AA64ISAR0_SHA2_FEAT_SHA256	U(1)
 #define ID_AA64ISAR0_SHA2_FEAT_SHA512	U(2)
 
+/* Pauth keys change from atomic ones to thread ones during SP initialization.
+ * Pointer authentication must be disabled during this time or the functions
+ * will not be able to return.
+ */
+#if (defined(CFG_CORE_PAUTH) || defined(CFG_TA_PAUTH))
+	#if (defined(CFG_CORE_BTI) || defined(CFG_TA_BTI))
+		#define DISABLE_PAUTH __attribute__((target("branch-protection=bti")))
+	#else
+		#define DISABLE_PAUTH __attribute__((target("branch-protection=none")))
+	#endif
+#else
+	#define DISABLE_PAUTH
+#endif
```

Comment:
prefer without the indentation.
can remove the outer parentheses at lines 312 and 313.

Reply:
Fixed